company_linkage/sql/selecting_top_conference_pubs.sql

```diff
@@ -31,26 +34,28 @@ SELECT
       OR REGEXP_CONTAINS(source_name, r'(?i)\bNeurIPS\b')
     OR REGEXP_CONTAINS(source_name, r'(?i)Annual Meeting of the Association for Computational Linguistics')
       OR ( REGEXP_CONTAINS(source_name, r'\bACL\b')
-      AND conference = 1
+      AND is_conference IS TRUE
     AND NOT REGEXP_CONTAINS(source_name, r'(?i)(injur)|(special interest)|(coling)'))
     OR REGEXP_CONTAINS(source_name, r'(?i)North American Chapter of the Association for Computational Linguistics')
          OR REGEXP_CONTAINS(source_name, r'(?i)\bNAACL\b')
-    OR REGEXP_CONTAINS(source_name, r'(?i)Conference on Empirical Methods in Natural Language Processing')
-      OR REGEXP_CONTAINS(source_name, r'(?i)\bEMNLP\b')
+    OR (REGEXP_CONTAINS(source_name, r'(?i)Conference on Empirical Methods in Natural Language Processing')
+      OR REGEXP_CONTAINS(source_name, r'(?i)\bEMNLP\b'))
+      AND NOT REGEXP_CONTAINS(source_name, r'(?i)SIGDAT')
```

Comment:
Did you intend to have parentheses around lines 41 through 43 so sigdat is only excluded when one of the other two regex match? (I don't know the details of emnlp, so this naive question is only motivated by the way the query is indented!)